addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -661,7 +668,10 @@ public void updateAvaliabilityStatus(IZoneMinderConnectionInfo connection) {
                     statusDetail = ThingStatusDetail.COMMUNICATION_ERROR;
                     statusDescription = "Session lost connection to ZoneMinder Server";
                     updateBridgeStatus(newStatus, statusDetail, statusDescription);
-                    logger.error("{}: {}", getLogIdentifier(), statusDescription);
+
+                    if (curBridgeStatus != ThingStatus.OFFLINE) {
+                        logger.error("{}: {}", getLogIdentifier(), statusDescription);
```

Comment:
You should actually not log an error at all - the OFFLINE status will already be logged as an event with the statusDescription.

Same for all other occurrences below.